content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md

```diff
@@ -92,5 +92,34 @@ Once there is information in the knowledge base, the agent will start using this
     <p style="margin-top: 20px; font-size: 15px">URLs added to Agent's Answer</p>
 </div>
 
+## URL Buttons from Spreadsheets
+
+You can also add URL buttons to your agent's responses using a spreadsheet. When you are adding knowledge using a spreadsheet to upload a large amount of information, it can be difficult to add reference buttons to each knowledge manually. Don't worry. SeaChat has a solution for you. 
+
+By adding an addtional column called **seachat_ref** in your spreadsheet, you agent will automatically extract information to add URL buttons to the agent's responses.
```

Comment:
users can add as many columns of "seachat_ref" as they want. THis sounds like they can only add one.